consoleme/lib/aws.py

```diff
@@ -1506,3 +1508,114 @@ async def get_scps_for_account_or_ou(identifier: str) -> ServiceControlPolicyArr
                 scps_for_account.append(scp)
     scps = ServiceControlPolicyArrayModel(__root__=scps_for_account)
     return scps
+
+
+async def minimize_iam_policy_statements(
```

Comment:
Let's add some tests for this since it could be security-impacting.

Reply:
It's covered indirectly, I'll see about adding some unit tests specific to this function

![image](https://user-images.githubusercontent.com/7538233/124002899-26d35e00-d98b-11eb-96d3-55fe9b6ea3ca.png)